frontend/packages/console-dynamic-plugin-sdk/src/extensions/cluster-overview.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { CodeRef, Extension, ExtensionDeclaration } from '../types';
+
+export type ClusterOverviewInventoryItem = ExtensionDeclaration<
```

Comment:
Please add a brief one-line JSDoc block describing the purpose of this extension :smiley:

Reply:
added :)